### DISCUSSION TITLE
Make ItyFuzz Focus on monatery exploits.

### DISCUSSION THREAD

**Discussion by ndtrung00** [fuzzland/ityfuzz#348] [Q&A]
I notice, there are a few exploited vulnerabilities have type of "ERC20", by which the monetary exploit is shown, something like: 

> "Earned {} more than owed {}, net earned = {}wei ({}ETH)\n"

I have current have 2 questions:
1. Do you guys have a explicit docs about which vulnerability the "bug_type" represent?
- "imbalanced_pair”
- "state_comp”
- "selfdestruct”
- "reentrancy”
- "invariant”
- "erc20”
- "echidna”
- "Arbitrary Transfer”
- "Arbitrary Call”
--> some are not so obvious by just reading the type, that's why I am just wondering a bit.

2. Is it possible to make ItyFuzz focus on just the monetary exploits like the "ERC20" type? if yes, which part of the code I should look into? Any lead is very appreciated! Feel free also to criticize the question if my question is fundamentally nonsense, I'm very curious about ItyFuzz and wanted to understand it a bit deeper.

Looking forwards to your response!

**Comment by shouc** [comment]
Working on it. We'll update the docs, and add CLI flags to disable some oracles. In the meantime, you can use `--run-forever` to let ItyFuzz not stop at a specific vulnerability and find all vulnerabilities
